.github/workflows/ci.yml

```diff
@@ -337,24 +344,48 @@ jobs:
           - target: x86_64-unknown-linux-gnu
             os: ubuntu-latest
     steps:
-      - name: Install and run Postgis
-        uses: nyurik/action-setup-postgis@v2
+      - name: Install and run Postgis (Windows)
+        if: matrix.os == 'windows-latest'
+        uses: nyurik/action-setup-postgis@v2.2
         id: pg
-        with: { username: 'test', password: 'test', database: 'test' }
+        with: { username: 'postgres', password: 'postgres', database: 'test',postgres-version: 14, port: 34837, postgis_version: 3.3.3}
+      - name: Install and run Postgis (Ubuntu)
```

Comment:
To be on the same page on this and not loose why  this code exists to time:
Why did you split it here? 🤔

Reply:
* With `services` we could set up various version of PostGIS/Postgres very easily, but only Linux runner support it..
* Setting PostGIS version is only supported for Windows runner by now with nyurik/action-setup-postgis@v2.2. I'm working on for [macos](https://github.com/nyurik/action-setup-postgis/pull/15)

Reply:
Sometimes I doubt maybe test only on Ubuntu is enough...